number/src/serialize.rs

```diff
@@ -74,4 +153,25 @@ mod tests {
         assert_eq!(read_polys, polys);
         assert_eq!(read_degree, degree as u64);
     }
+
+    #[test]
+    fn write_read_csv() {
+        let polys = test_polys()
+            .into_iter()
+            .map(|(name, values)| (name.to_string(), values))
+            .collect::<Vec<_>>();
+
+        for render_mode in &[
+            CsvRenderMode::SignedBase10,
+            CsvRenderMode::UnsignedBase10,
+            CsvRenderMode::Hex,
+        ] {
+            println!("render_mode: {:?}", render_mode);
```

Comment:
is this print gonna stay?

Reply:
No, wasn't on purpose! Guess the linter rule does not apply to test modules?